### PR TITLE
StandardDialog: make scrollable + close button

### LINKED ISSUE #1156: Can't scroll down on the confirmation window
The confirmation window is borked, i.e., it doesn't allow you to scroll down and see part of the transaction you are going to send.

### PR DESCRIPTION
**Pull request by mmbyday**
fixes #1156
asfsdf

### CODE REVIEW

File: components/StandardDialog.qml [hunk 1]
```diff
@@ -89,55 +89,66 @@ Rectangle {
...
-    width: isMobile ? screenWidth : 520
-    height: isMobile ? screenHeight : 380
+    width: isMobile ? screenWidth : 520 * scaleRatio
+    height: isMobile ? screenHeight : 380 * scaleRatio
 
     ColumnLayout {
         id: mainLayout
         spacing: 10
-        anchors { fill: parent; margins: 15 }
+        anchors.margins: (isMobile? 17 : 20) * scaleRatio
```
Comment (xiphon) [inline components/StandardDialog.qml]:
```diff
@@ -95,11 +96,8 @@ Rectangle {
     ColumnLayout {
         id: mainLayout
         spacing: 10
+        anchors.fill: parent
         anchors.margins: (isMobile? 17 : 20) * scaleRatio
-        Layout.fillWidth: true
-        anchors.left: parent.left
-        anchors.top: parent.top
-        anchors.right: parent.right
 
         RowLayout {
             id: column
```

File: components/StandardDialog.qml [hunk 2]
```diff
@@ -160,44 +171,79 @@ Rectangle {
...
+                }
+            }
+        }
+    }
+
+    Row {
+        id: row
+        anchors.right: parent.right
+        anchors.top: parent.top
+        visible: parent.customDecorations
```
Comment (xiphon) [inline components/StandardDialog.qml]: * Drop `Row`, keep `Rectangle`
* Remove `z` property
* Replace `visible: parent.customDecorations` with `property alias closeVisible: closeButton.visible`

File: components/StandardDialog.qml [hunk 3]
```diff
@@ -160,44 +169,77 @@ Rectangle {
...
                 KeyNavigation.tab: cancelButton
                 onClicked: {
                     root.close()
                     root.accepted()
+                }
+            }
+        }
+    }
 
+    Rectangle {
```
Comment (xiphon) [inline components/StandardDialog.qml]:
```diff
diff --git a/components/StandardDialog.qml b/components/StandardDialog.qml
index b388fe8..15069a3 100644
--- a/components/StandardDialog.qml
+++ b/components/StandardDialog.qml
@@ -179,37 +179,31 @@ Rectangle {
         }
     }
 
+    // close icon
     Rectangle {
-        id: buttonRect
-        anchors.right: parent.right
+        id: closeButton
         anchors.top: parent.top
+        anchors.right: parent.right
+        width: 48 * scaleRatio
+        height: 48 * scaleRatio
+        color: "transparent"
+
+        Image {
+            anchors.centerIn: parent
+            width: 16 * scaleRatio
+            height: 16 * scaleRatio
+            source: "../images/close.png"
+        }
 
-        // close icon
-        Rectangle {
-            id: closeButton
-            anchors.top: parent.top
-            anchors.right: parent.right
-            width: 48 * scaleRatio
-            height: 48 * scaleRatio
-            color: "transparent"
-
-            Image {
-                anchors.centerIn: parent
-                width: 16 * scaleRatio
-                height: 16 * scaleRatio
-                source: "../images/close.png"
-            }
-
-            MouseArea {
-                anchors.fill: parent
-                onClicked: {
-                    root.close()
-                    root.rejected()
-                }
-                cursorShape: Qt.PointingHandCursor
-                onEntered: closeButton.color = "#262626";
-                onExited: closeButton.color = "transparent";
+        MouseArea {
+            anchors.fill: parent
+            onClicked: {
+                root.close()
+                root.rejected()
             }
+            cursorShape: Qt.PointingHandCursor
+            onEntered: closeButton.color = "#262626";
+            onExited: closeButton.color = "transparent";
         }
     }
```

Comment (mmbyday) [reply components/StandardDialog.qml]: Updated. Thank you.

**Pull request merged**